docs/nx-cloud/reference/launch-templates.md

```diff
@@ -1,42 +1,296 @@
 # Launch Templates
 
-You can find built-in launch templates [here](https://github.com/nrwl/nx-cloud-workflows/tree/main/launch-templates).
+A launch template defines the setup steps Nx Agents will run before running tasks. A custom launch template isn't required to use Nx Agents.
+By default, Nx provides a simplified one for you. You can find built-in launch templates in the [`nx-cloud-workflows` repository](https://github.com/nrwl/nx-cloud-workflows/tree/main/launch-templates).
+
+{% github-repository url="https://github.com/nrwl/nx-cloud-workflows/tree/main/launch-templates" title="Built-in Launch Templates" /%}
 
 The easiest way to create a new custom launch template is to modify one of the built-in ones. To do that, create a file in the
-`.nx/workflows` folder and copy one of the built-in templates there. You can name the file any way you want (e.g., `agents.yaml`).
+`.nx/workflows` folder and copy one of the [built-in templates](https://github.com/nrwl/nx-cloud-workflows/blob/main/launch-templates/linux.yaml). You can name the file any way you want (e.g., `agents.yaml`) and customize the steps as needed.
 
 This is an example of a launch template using all built-in features:
 
-```yaml
+```yaml {% fileName="./nx/workflows/agents.yaml" %}
```

Comment:
I made this template way longer, which might be a lot at first, but wanted to provide a semi decent example file for people to build off with lots of comments.